src/main/java/com/hillel/items_exchange/controller/CategoryController.java

```diff
@@ -21,86 +24,63 @@
 
 @RestController
 @RequestMapping("/category")
-@CrossOrigin(origins = "http://localhost:4200")
 @RequiredArgsConstructor
-@Slf4j
 public class CategoryController {
 
     private final CategoryService categoryService;
 
     @GetMapping("/names")
-    public ResponseEntity<List<String>> getAllCategoriesNames() {
-        List<String> categoriesNames = categoryService.findAllCategoryNames();
-        if (categoriesNames.isEmpty()) {
-            return new ResponseEntity<>(HttpStatus.NOT_FOUND);
-        }
-
-        return new ResponseEntity<>(categoriesNames, HttpStatus.OK);
+    @ApiOperation(value = "Get all names of existing categories.")
+    public ResponseEntity<List<String>> getAllCategoryNames() {
+        return new ResponseEntity<>(categoryService.findAllCategoryNames(), HttpStatus.OK);
```

Comment:
If you response here with only one code you could use  ``@ResponseStatus(HttpStatus.OK)`` and return List<String>.

Reply:
Amazing!